conans/client/manager.py

```diff
@@ -492,7 +492,19 @@ def _get_search_adapter(self, remote):
 
         return adapter
 
+    def search_recipes_all(self, pattern, ignorecase):
+        reg = RemoteRegistry(self._client_cache.registry, self._user_io.out)
+        references = {}
+        for remote in reg.remotes:
+            remote_proxy = ConanProxy(self._client_cache, self._user_io, self._remote_manager, remote.name)
+            result = remote_proxy.search(pattern, ignorecase)
+            if result:
+                references[remote.name] = result
+        return references
+
     def search_recipes(self, pattern, remote, ignorecase):
+        if remote == 'all':
```

Comment:
What happened with the remote named "all" case?

Reply:
That case would be "broken" in the sense that it will search in all remotes, including the "all" remote